packages/react-native/src/Authenticator/hooks/useFieldValues/utils.ts

```diff
@@ -77,6 +75,36 @@ export const getSanitizedRadioFields = (
   });
 };
 
+export const getSanitizedFields = (fields: TypedField[]): TypedField[] => {
+  const names: Record<string, boolean> = {};
+  return fields.filter((field) => {
+    const { name } = field;
+
+    if (!name) {
+      logger.warn('Each field must have a name; field has been ignored.');
+      return false;
+    }
+
+    if (names[name]) {
+      logger.warn(
+        `Each field name must be unique; field with duplicate name of ${name} has been ignored.`
+      );
+      return false;
+    }
+
+    if (isRadioFieldOptions(field) && !field.radioOptions?.length) {
+      logger.warn(
+        `Each radio field must have at least one option available for selection; field without radioOptions ${name} has been ignored.`
```

Comment:
Nit: Kinda reads like the radioOptions are named `name`. Is that intentional or should we consider maybe moving name closer to field or just adding parentheses around `${name}`?